docs/maintainers/issue-management.md

```diff
@@ -53,15 +53,15 @@ These labels describe what feature area of PowerShell that an issue affects:
 * `Area-HelpSystem`: anything related to the help infrastructure and formatting of help
 * `Area-Intellisense`: tab completion
 * `Area-Language`: parser, language semantics
-* `Area-OMI`: omi
+* `Area-OMI`: OMI
 * `Area-PackageManagement`: PackageManagement related issues
 * `Area-Performance`: a performance issue
 * `Area-Portability`: anything affecting script portability
 * `Area-PowerShellGet`: PowerShellGet related issues
-* `Area-Providers`: PowerShell providers like FileSystem, Certificates, Registry, etc...
+* `Area-Providers`: PowerShell providers such as FileSystem, Certificates, Registry, etc...
```

Comment:
@sdwheeler 	Do you have any comments on which is more appropriate here?

Reply:
`like` is not very professional sounding.  `such as` just sounds better in my opinion.

Reply:
Either way is fine. No preference.